CITATION.cff

```diff
@@ -615,6 +615,10 @@ authors:
     family-names: Jain
     website: https://github.com/PrakharJain1509
     affiliation: RV College of Engineering, Bangalore, India
+  - given-names: quentden
```

Comment:
@quentden I would need you to update those info to be listed in the contributors